plugins/database/redshift/redshift_test.go

```diff
@@ -48,38 +47,40 @@ var (
 	vaultACC = "VAULT_ACC"
 )
 
-func redshiftEnv() (url string, user string, password string, errEmpty error) {
+func interpolateConnectionURL(url, user, password string) string {
+	return fmt.Sprintf("postgres://%s:%s@%s", user, password, url)
+}
+
+func redshiftEnv() (connURL string, url string, user string, password string, errEmpty error) {
 	errEmpty = errors.New("err: empty but required env value")
 
 	if url = os.Getenv(keyRedshiftURL); url == "" {
-		return "", "", "", errEmpty
+		return "", "", "", "", errEmpty
 	}
 
 	if user = os.Getenv(keyRedshiftUser); url == "" {
-		return "", "", "", errEmpty
+		return "", "", "", "", errEmpty
 	}
 
 	if password = os.Getenv(keyRedshiftPassword); url == "" {
-		return "", "", "", errEmpty
+		return "", "", "", "", errEmpty
 	}
 
-	url = fmt.Sprintf("postgres://%s:%s@%s", user, password, url)
-
-	return url, user, password, nil
+	return interpolateConnectionURL(url, user, password), url, user, password, nil
```

Comment:
nit: can `interpolateConnectionURL` be split out into its own call? This line is hard to read